achievements/tests/tests_serializers.py

```diff
@@ -27,7 +27,7 @@ def test_serializer_claimant(self):
             "extra_data_2":     "xyz",
             "extra_data_3":     "equal",
         })
-    
```

Comment:
Do we have a default formatter?

Then we can run that, add a test to ci with the formatter, and not have things like this anymore.

Then we don't get problems like this.

Reply:
We've introduced a [`.editorconfig`](https://github.com/esrg-knights/Squire/blob/master/.editorconfig) at some point, but that wasn't there from the start. Changes like these basically mean that this file hasn't been touched since we introduced the editorconfig. Some IDEs support this out of the box, while others require an extension (like VSCode)

We haven't re-saved all the files in the repo to fix these issues in a single go, so that could be worthwhile to do.